doc/api/process.md

```diff
@@ -1429,15 +1428,20 @@ Android)
 The `process.stderr` property returns a [Writable][] stream equivalent to or
 associated with `stderr` (fd `2`).
 
-`process.stderr` and `process.stdout` are unlike other streams in Node.js in
-that they cannot be closed (calling [`end()`][] will throw an Error), they never
-emit the [`'finish'`][] event, and writes can block when output is redirected to
-a file (although disks are fast and operating systems normally employ write-back
-caching so it should be a very rare occurrence indeed.)
+Note: `process.stderr` and `process.stdout` are differ from other streams in
+Node.js in a couple ways:
+1. They cannot be closed ([`end()`][] will throw).
+2. They never emit the [`'finish'`][] event.
+3. Writes _can_ block when output is redirected to a file.
+  - (Although disks are fast and operating systems normally employ write-back
```

Comment:
this doesn't make sense bracketed as a lone sub-point, remove sub-point and bring it back up to the previous line, I'd even remove the brackets but that up to you.